From fe00f8cc3dbd419ff4de11d0dde3bd6a16052fc7 Mon Sep 17 00:00:00 2001
From: thongsyho <thong.ho.px@rvc.renesas.com>
Date: Mon, 22 Jul 2019 11:22:15 +0700
Subject: [PATCH 364/442] ARM: dts: r8a77470: Add VSP support to SoC dtsi

Add Video Signal Processing (vsp) nodes to device tree of r8a77470.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index 3d9095f..70141e2 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -842,6 +842,22 @@
 			status = "disabled";
 		};
 
+		vspm@fe928000 {
+			compatible = "renesas,vspm-vsps", "renesas-vspm";
+			reg = <0 0xfe928000 0 0x7404>;
+			interrupts = <0 267 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A77470_CLK_VSP1_SY>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
+		};
+
+		vspm@fe930000 {
+			compatible = "renesas,vspm-vspd0", "renesas-vspm";
+			reg = <0 0xfe930000 0 0x7404>;
+			interrupts = <0 246 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A77470_CLK_VSP1DU0>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
+		};
+
 		sdhi0: mmc@ee100000 {
 			compatible = "renesas,sdhi-r8a77470",
 				     "renesas,rcar-gen2-sdhi";
-- 
2.7.4

